### PR TITLE
Rename LTP variables with prefix for the corresponding module

### PR DESCRIPTION
**Pull request by b10n1k**
following the name convention that kernel modules uses and for the sake of sanity between those which are used for the same exact purpose.
that also will make the variable more specific. for instance
COMMAND_EXCLUDE - LTP_COMMAND_EXCLUDE
COMMAND_FILE - LTP_COMMAND_FILE

- Related ticket: https://progress.opensuse.org/issues/98334
- Verification run: https://openqa.suse.de/t7719617
https://openqa.suse.de/tests/7725754 for d7c99b0d659e9e17ae910bd4f6848adb72d631d9

### CODE REVIEW

**Review by grisu48 — APPROVED** [review]
LGTM :rocket: 

I'd like to see one VR with a intact job schedule, then we can merge. As long as LTP runs, it should be safe, IMHO we don't need to wait for a full run.

**Review by pdostal — APPROVED** [review]
:rocket:

**Review by asmorodskyi — APPROVED** [review]
LGTM , can be merged but after VR

**Review by jlausuch — APPROVED** [review]

**Comment by czerw** [comment]
We use `LTP_COMMAND_FILE` and `LTP_COMMAND_EXCLUDE` already and changed files are use by public cloud only. It seems ok for.  But better doublecheck @mdoucha @pevik could  you please review as well?

File: variables.md [hunk 1]
```diff
@@ -96,6 +94,8 @@ LIVECD | boolean | false | Indicates live image being used.
 LIVE_INSTALLATION | boolean | false | If set, boots the live media and starts the builtin NET installer.
 LIVE_UPGRADE | boolean | false | If set, boots the live media and starts the builtin NET installer in upgrade mode.
 LIVETEST | boolean | false | Indicates test of live system.
+LTP_COMMAND_FILE | string | | The LTP test command file (e.g. syscalls, cve)
+LTP_COMMAND_EXCLUDE | string | | This regex is used to exclude tests from LTP command file.
```
Comment (pevik) [inline variables.md:98]: LGTM. FYI we document these variables already in tests/kernel/run_ltp.pm. Also there are more variables, LTP_COMMAND_PATTERN in "our" LTP scripts. Not sure if it's a good idea to document some variables here and some in test modules.

**Review by pevik — APPROVED** [review]

**Review by czerw — APPROVED** [review]
LGTM

File: variables.md (hunk 1 above)
Comment (jlausuch) [reply variables.md:98]: I would favor for `variables.md`, which is more visible.

**Pull request merged**